samples/Samples.AspNetCoreMvc2/Samples.AspNetCoreMvc2.csproj

```diff
@@ -14,7 +17,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore" Version="2.0.4" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.0.4" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="$(ApiVersion)" />
```

Comment:
Should we match the versions of `Microsoft.AspNetCore.Mvc` and `Microsoft.AspNetCore`?

Reply:
There isn't a strict 1:1 correspondence between the two packages (e.g. there are more package versions of `Microsoft.AspNetCore` than `Microsoft.AspNetCore.Mvc`) so I decided to lock `Microsoft.AspNetCore`. However, upon closer inspection, it is the case that for every `Microsoft.AspNetCore.Mvc` there is a package of `Microsoft.AspNetCore` with the same version number. So perhaps I should just set both to `$(ApiVersion)`, what do you think?

Alternatively, except for one package, each version of `Microsoft.AspNetCore.Mvc` has a corresponding `Microsoft.AspNetCore` package that was published on the same day. Perhaps I can try to find that match? That would definitely be the more complicated route and I'm not sure how much benefit it brings us.

Reply:
I've set both to `$(ApiVersion)`, which should make the testing a little more diverse.